cypress/e2e/CMS/$CancerCenter.feature

```diff
@@ -0,0 +1,153 @@
+Feature: As a cms user I want to be able to create cancer center content type to promote cancer centers
+
+    Scenario: Verify all the fields on the cancer center content creation page
```

Comment:
I verbally said this on the phone, but putting this here. 

Kate pointed out the reason for this test is that the "normal" test does not touch all fields. This test makes sure they are all here.

I noted that we should not do this, and instead should have a test that actually *uses* all the fields. A "kitchen" sink test if you will. See the fields does not test that the field works correctly.

Reply:
I am ok for right now if we keep it as it is in this PR, as it is better to automate the normal tests and reduce that effort each upgrade. We should put the kitchen tests in the backlog for increasing our coverage.